dynamic-property-zk-test/src/main/java/ru/fix/dynamic/property/zk/test/SocketChecker.java

```diff
@@ -0,0 +1,54 @@
+package ru.fix.dynamic.property.zk.test;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.DatagramSocket;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class SocketChecker {
+
+    private static final Logger log = LoggerFactory.getLogger(SocketChecker.class);
+
+    private static final int PORTS_FROM = 30000;
+    private static final int PORTS_TO = 60000;
+
+    private SocketChecker() {
+    }
+
+    public static boolean isAvailable(int port) {
+        try (ServerSocket serverSocket = new ServerSocket(port);
+             DatagramSocket datagramSocket = new DatagramSocket(port)) {
+
+            return serverSocket.isBound() && datagramSocket.isBound();
+        } catch (Exception exc) {
+            /**
+             * ignore exception
```

Comment:
Socket checker will be in jfix-stdlib-socket library. @AnonymousVovus are currently working on moving this class there.

Reply:
add after that will be done

Reply:
`SocketChecker.java` moved to `jfix-stdlib-socket:1.0.22` and released. Waiting for maven.central...